lib/Client.js

```diff
@@ -2532,6 +2562,23 @@ class Client extends EventEmitter {
     }).then((dmChannel) => new DMChannel(dmChannel, this));
   }
 
+  /**
+   * Get an emoji for this application
+   * @arg {String} emojiID The ID of the emoji
+   * @returns {Promise<Object>} Resolves with an emoji object
+   */
+  getEmoji(emojiID) {
+    return this.requestHandler.request("GET", Endpoints.APPLICATION_EMOJI(this.application.id, emojiID), true);
+  }
+
+  /**
+   * Get the emojis for this application
+   * @returns {Promise<Array<Object>>} Resolves with an array of emoji objects
+   */
+  getEmojis() {
+    return this.requestHandler.request("GET", Endpoints.APPLICATION_EMOJIS(this.application.id), true);
```

Comment:
Update `user` cache. Also this doesn't directly return an emojis array, it returns an object containing `items`, which is the array of emojis